dfsclient/src/test/java/org/hpccsystems/dfs/client/DFSReadWriteTest.java

```diff
@@ -1039,7 +1041,6 @@ private void writeFile(List<HPCCRecord> records, String fileName, FieldDef recor
     public void readIndexTest() throws Exception
     {
         String indexName = "test::index::child_dataset::key";
-        executeECLScript("generate-nested-index.ecl");
```

Comment:
@rpastrana Yep, sorry forgot to mention that on this mornings call. I did some testing and found that the original plan wouldn't work as well as I had hoped. IE: I couldn't share ECL data generation scripts across projects so I would need to add a version of the generate-datasets.ecl scripts in each project either way. So instead of using a specialized subclass handle the each projects different data generation needs, I created a generate-datasets.ecl script for each separate project that handled generating the needed datasets for each project. This also cut down on the test execution time by around 5 min.